src/decorator/options/ColumnOptions.ts

```diff
@@ -149,6 +149,12 @@ export interface ColumnOptions extends ColumnCommonOptions {
      */
     hstoreType?: "object"|"string";
 
+    /**
+     * Return type of JSON/JSONB column.
+     * Returns value as string or as object.
```

Comment:
"Returns"? Based on issue description I thought we are talking about value we send to the database...

Reply:
This is analogous to the HSTORE type. Yes, this applies to both the value sent and value returned from the database.